scriptmodules/emulators/aethersx2.sh

```diff
@@ -0,0 +1,539 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="aethersx2"
+rp_module_desc="PS2 emu - AetherSX2"
+rp_module_help="ROM Extensions: .iso .chd\n\nCopy your PS2 roms to $romdir/ps2"
+rp_module_licence="PROP"
+rp_module_repo="git https://github.com/retropieuser/aethersx2.git main"
+rp_module_section="exp"
+rp_module_flags="!all 64bit"
+
+function depends_aethersx2() {
+    local depends=(matchbox-window-manager xorg xserver-xorg-input-all mesa-vulkan-drivers pulseaudio)
```

Comment:
```suggestion
    local depends=(matchbox-window-manager xorg xserver-xorg-input-all mesa-vulkan-drivers pulseaudio pipewire-media-session-pulseaudio)
```

with just ``pulseaudio`` still did not work, but with ``pipewire-media-session-pulseaudio`` added, i have sound and GTA plays and sounds nice

i dont use the pi imager, my work has that blocked from downloading the ``img`` file. 
i have to download it manually and then use etcher to burn the SD card. but its still pi OS lite 64bit

i dont think your way installs or includes any different drivers compared to mine